docs/SeismicLookupTable.md

```diff
@@ -0,0 +1,37 @@
+TerraTools allows for interrogation of seismic lookup tables by interpolation and plotting. Futhermore, TerraTools can convert terra output in temperature and composition to a seismic model for given look up tables for each composition present. 
+
+## Seismic lookup tables and how to read them in
+
+A seismic lookup table describes how seismic properties such as Vp (p-wave velocity) or density vary with pressure and temperature for a given composition. In TerraTools, you can make a SeismicLookupTable by either reading in a file from disk or passing arrays to the SeismicLookupTable class. 
+
+When reading in a lookup table file via ``SeismicLookupTable(table_path='path_to_file')``, the file needs to be in the format shown below. Note that the temperatures increase first and then the pressures. In other words, the values at all temperatures are given for each pressure point first, then all values for the temperatures at the second pressure point and so on. The table below gives an example of this. 
```

Comment:
You can create links to the docstring-derived reference section for a function using the `[][]` syntax.  E.g. here, you can do
```
`[SeismicLookupTable(table_path='path_to_file')][terratools.lookup_tables.SeismicLookupTable]`
```
so this becomes a link